test/test_cmath.rb

```diff
@@ -3,7 +3,16 @@
 require 'cmath'
 
 class TestCMath < Test::Unit::TestCase
+  def test_any_method!
```

Comment:
I'm okay with this test-case. But I hope to reason why the name of this test-case is `any_method!`. Can you elaborate on it?

Reply:
Sory. I given not good name to this method. 
Rename test that check deprecated method.

Reply:
Rename to `test_deprecated_method`, thanks.